manager/system_handler.py

```diff
@@ -134,7 +134,10 @@ class GetSystems(GetRequest):
     @classmethod
     def handle_get(cls, **kwargs):
         """Gets all systems and count of their vulnerabilities"""
-        args_desc = [{'arg_name': 'opt_out', 'convert_func': None}]
+        args_desc = [
+            {'arg_name': 'opt_out', 'convert_func': None},
+            {'arg_name': 'filter', 'convert_func': remove_str_nulls}
```

Comment:
where does filter come from? it wasn't in this API before

Reply:
Shouldn't be there